apps/omg_api/lib/block_queue/core.ex

```diff
@@ -333,7 +333,23 @@ defmodule OMG.API.BlockQueue.Core do
          },
          is_empty_block
        ) do
-    parent_height - last_enqueued_block_at_height > minimal_enqueue_block_gap and !wait_for_enqueue and !is_empty_block
+    it_is_time = parent_height - last_enqueued_block_at_height > minimal_enqueue_block_gap
+    should_form_block = it_is_time and !wait_for_enqueue and !is_empty_block
+
+    if !should_form_block do
+      log_data = %{
```

Comment:
Is this `log_data` used for logging only ❓ If so please move it inside `Logger.debug` f-ction. We will avoid memory allocation in higher log levels